components/MetroBarChart.vue

```diff
@@ -2,15 +2,48 @@
   <data-view :title="title" :date="date">
     <template v-slot:button>
       <p class="MetroGraph-Desc">
-        {{ chartData.base_period }}の利用者数*の平均値を100としたときの相対値
+        {{ $t('相対値', { range: chartData.base_period }) }}
```

Comment:
一応 https://github.com/tokyo-metropolitan-gov/covid19/issues/680#issuecomment-596056211 のとおり「翻訳文字列のキー＝日本語訳」にしたいので
```suggestion
        {{ $t('{range}の利用者数*の平均値を100としたときの相対値', { range: chartData.base_period }) }}
```にしてもらいたいのですが、そもそもこの指定方法で動きますかね...？

Reply:
@MaySoMusician ちょっと試してみます